chtransmissionformat/README.md

```diff
@@ -0,0 +1,267 @@
+# Specification ChTransmissionFormat
+
+**Contact**
+
+Geschäftsstelle IG eMediplan<br>
+c/o Köhler, Stüdeli & Partner GmbH<br>
+Amthausgasse 18, 3011 Bern<br>
+Tel. +41 (0)31 560 00 24<br>
+info@emediplan.ch
+
+## Introduction
+
+Medication plans often need to be printed on paper.
+To easily bring these back into the digital world,
+the CHMED standard (in all its versions) requires a specification
+of how the whole information can be represented in a QR code
+as a simple line of text.
+
+The ChTransmissionFormat is this corresponding format
+which defines the contents of the QR code on a printed medication plan
+(which can be represented either physically on paper or digitally e.g. as a PDF).
+This representation can be used anywhere
+but is optimized for the usage in the QR code.
+
+### Goals
+
+- Define parsing rules based on prefix of the data
+  (only looking at the start of the data needs to be sufficient
+  to determine how the rest needs to be parsed),
+  specifically the version part
+  which allows to implement support for multiple versions.
+- Define measures to reduce the size of data since the amount of data
+  which can be included in a single QR code is limited.
+
+## Format
+
+A ChTransmissionFormat document must use UTF-8 encoding.
+
+```
+CHMED{ReleaseYear}{SubVersion}{AdditionalMetaData}{Data}
+```
+
+Representation as regex:
+
+```regex
+^CHMED([0-9]{2})([A-Z]+)(.*)$
+```
+
+Description and constraints:
+
+- `CHMED` (Prefix): This is the fixed string `CHMED`.
+  It is always the same value and should be used to determine if the given input string
+  should be tried to parse as ChTransmissionFormat.
+  During parsing, it should be treated as case-insensitive (e.g. `ChMed` should also be accepted)
+  but during serialization all uppercase should be used.
```

Comment:
You're using `should` a lot here. It is actually just a recommendation or do you mean `must`/`shall`?
Good reference: https://www.ietf.org/rfc/rfc2119.txt

Reply:
Very good reference, I think I'll incorporate the proposed header of rfc2119 and will review the places where I use the keywords.